### PR TITLE
Update README.md - Community Integrations - vscode, Sublime Text, CLI…

### PR DESCRIPTION
**Pull request by icebaker**
:wave: I have added new integrations for CLI, Ruby, Visual Studio Code, Sublime Text, and Obsidian.

*VSCode Demonstration:

https://github.com/jmorganca/ollama/assets/113217272/e6ba9c62-56d5-401f-8b63-51407d9154bd

*CLI Demonstration:

https://github.com/jmorganca/ollama/assets/113217272/5612653b-c279-4fe7-910f-f734e26f4489

> _* The videos were edited: Typing speed accelerated by 1.5x, the delay before streaming was cut out, and the answers were accelerated by 4x._

- [Nano Bots CLI](https://github.com/icebaker/ruby-nano-bots)
- [Nano Bots for Ruby](https://github.com/icebaker/ruby-nano-bots)
- [Visual Studio Code](https://github.com/icebaker/vscode-nano-bots)
- [Sublime Text](https://github.com/icebaker/sublime-nano-bots)
- [Obsidian](https://github.com/icebaker/obsidian-nano-bots)

### CODE REVIEW

**Comment by mchiang0610** [comment]
Thanks @icebaker possible to say it's Nano Bots for VSCode, Sublime Text, and Obsidian. I just don't want to cause user confusion that it's a direct integration from the respective application owners.

File: README.md [hunk 1]
```diff
@@ -276,6 +276,7 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [gptel Emacs client](https://github.com/karthink/gptel)
 - [Oatmeal](https://github.com/dustinblackman/oatmeal)
 - [cmdh](https://github.com/pgibler/cmdh)
+- [Nano Bots CLI](https://github.com/icebaker/ruby-nano-bots)
```
Comment (jmorganca) [inline README.md:279]: This is super cool! Would it be possible to only add this once just since it's the same URL in both cases

File: README.md [hunk 2]
```diff
@@ -320,3 +322,6 @@ See the [API documentation](./docs/api.md) for all endpoints.
 - [Rivet plugin](https://github.com/abrenneke/rivet-plugin-ollama)
 - [Llama Coder](https://github.com/ex3ndr/llama-coder) (Copilot alternative using Ollama)
 - [Obsidian BMO Chatbot plugin](https://github.com/longy2k/obsidian-bmo-chatbot)
+- [Obsidian Nano Bots plugin](https://github.com/icebaker/obsidian-nano-bots)
```
Comment (jmorganca) [inline README.md:325]: Would it be possible to add this as one entry? Thanks!

**Comment by jmorganca** [comment]
Hi @icebaker thanks so much for the PR. I haven't heard back for a while and and so with the goal of cleaning up open PRs I'll close this for now. However let me know if you'd like to add the project (it looks really cool) and we can re-open (or feel free to create a new PR!)

**Pull request closed without merging**